linked_list/linked_list.cpp

```diff
@@ -0,0 +1,330 @@
+/*
+* C++ Program to Implement Linked List
+*/
+#include <iostream>
+#include <cstdio>
+#include <cstdlib>
+
+using namespace std;
+/*
+* Node Declaration
+*/
+class node
+{
```

Comment:
@S2606 Please follow Google C++ style guide for comments and curly braces( `{` ).
https://google.github.io/styleguide/cppguide.html#Class